examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,231 @@ Documentation for other releases can be found at
+```shell
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/gce-volumes.yaml
 ```
 
-Then create the wordpress disk.
+### Host Path
 
-```sh
-gcloud compute disks create --size=20GB --zone=$ZONE wordpress-disk
-```
+Host paths are volumes mapped to directories on the host. These should
+be used for testing or single-node clusters only, as the data will not
+be moved between nodes if the pod is recreated on a new node. Create
+the persistent volume objects in Kubernetes using
+[local-volumes.yaml](local-volumes.yaml):
 
-## Start the Mysql Pod and Service
-
-Now that the persistent disks are defined, the Kubernetes pods can be launched.  We'll start with the mysql pod.
-
-### Start the Mysql pod
-
-First, **edit [`mysql.yaml`](mysql.yaml)**, the mysql pod definition, to use a database password that you specify.
-`mysql.yaml` looks like this:
-
-<!-- BEGIN MUNGE: EXAMPLE mysql.yaml -->
-
-```yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  name: mysql
-  labels: 
-    name: mysql
-spec: 
-  containers: 
-    - resources:
-        limits :
-          cpu: 0.5
-      image: mysql:5.6
-      name: mysql
-      env:
-        - name: MYSQL_ROOT_PASSWORD
-          # change this
-          value: yourpassword
-      ports: 
-        - containerPort: 3306
-          name: mysql
-      volumeMounts:
-          # name must match the volume name below
-        - name: mysql-persistent-storage
-          # mount path within the container
-          mountPath: /var/lib/mysql
-  volumes:
-    - name: mysql-persistent-storage
-      gcePersistentDisk:
-        # This GCE PD must already exist.
-        pdName: mysql-disk
-        fsType: ext4
+```shell
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/local-volumes.yaml
 ```
 
-[Download example](mysql.yaml?raw=true)
-<!-- END MUNGE: EXAMPLE mysql.yaml -->
-
-Note that we've defined a volume mount for `/var/lib/mysql`, and specified a volume that uses the persistent disk (`mysql-disk`) that you created.
-Once you've edited the file to set your database password, create the pod as follows, where `<kubernetes>` is the path to your Kubernetes installation:
-
-```sh
-$ kubectl create -f examples/mysql-wordpress-pd/mysql.yaml
-```
+## Create the MySQL Password Secret
 
-It may take a short period before the new pod reaches the `Running` state.
-List all pods to see the status of this new pod and the cluster node that it is running on:
+Use a [Secret](../../docs/user-guide/secrets.md) object to store the
+MySQL password. This is referenced by the MySQL and WordPress pods when
+they are created. First create a temporary file called `password.txt`
+and save your password in it. Make sure to not have a trailing newline
+at the end of the password. Then, create the Secret object.
 
-```sh
-$ kubectl get pods
+```shell
+kubectl create secret generic mysql-pass --from-file=password.txt
```

Comment:
Explain this command.  What are the `generic` and `mysql-pass` arguments defining?